lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -4,8 +4,8 @@ use Bio::KBase::Exceptions;
 # Use Semantic Versioning (2.0.0-rc.1)
 # http://semver.org
 our $VERSION = '0.1.6';
```

Comment:
this needs to be updated

Reply:
I agree, I just noticed my PR #82 was closed which has the change of version number.